lighthouse-core/report/html/templates.html

```diff
@@ -164,11 +164,13 @@
       width: 100%;
       will-change: transform;
     }
+    .lh-narrow .lh-header-bg {
+      height: var(--report-header-height-narrow);
```

Comment:
eh. i dont think we need this. the jump is a bit awkward and putting the LH behind the text isn't terrible:
![image](https://user-images.githubusercontent.com/39191/50619144-58e3d200-0eac-11e9-872e-328193d77064.png)

but if you wanted to avoid that.. we could do `  max-width: 60%;` on `.lh-config`
![image](https://user-images.githubusercontent.com/39191/50619178-8597e980-0eac-11e9-9322-c552193bbf4c.png)